notebooks/en/issues_in_text_dataset.ipynb

```diff
@@ -0,0 +1,3635 @@
+{
```

Comment:
<p>Please use the actual name and not the account handle for the author, for consistency with other notebooks, i.e. </p><pre class="ql-syntax" spellcheck="false">[FirstName LastName](link_to_HF_profile)
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>